src/common/logger.cpp

```diff
@@ -6,11 +6,20 @@
 
 #include "common/logger.hpp"
 
+#include <csignal>
 #include <cstdlib>
 
 #include "axom/slic.hpp"
 #include "mpi.h"
 
+namespace {
```

Comment:
I'm unfamiliar with this use of an unnamed namespace, what is it doing?

Reply:
It makes sure that it isn't available for external linkage - I use it for little utility functions, mostly.  It's equivalent to declaring the function `static`.

The Google C++ style guide has a [little blurb on it](https://google.github.io/styleguide/cppguide.html#Unnamed_Namespaces_and_Static_Variables) as well.

Reply:
If the `static` keyword is more readable I'll switch it

Reply:
I have no objection, just hadn't seen that before. Thanks for the explanation!